packages/desktop-client/src/components/common/Select.tsx

```diff
@@ -49,13 +49,6 @@ export default function Select<Value extends string>({
   disabledKeys = [],
 }: SelectProps<Value>) {
   const arrowSize = 7;
-  const checkHeight = !style
```

Comment:
Changes in this file are style reverts to the `Select` component introduced here https://github.com/actualbudget/actual/commit/9fed15f88b0e124cfbd663a4b233ee32c12ec02f#diff-5f4290e10caa20f2b8c41fae4d8f3b0aa45662d695adab848fd19381279f5e5a

I did not revert the colors, so those should still be fine in darkmode.